packages/jsii-java-runtime-test/project/src/test/java/software/amazon/jsii/testing/ComplianceTest.java

```diff
@@ -4,69 +4,14 @@
-import software.amazon.jsii.tests.calculator.Constructors;
-import software.amazon.jsii.tests.calculator.DataRenderer;
-import software.amazon.jsii.tests.calculator.DerivedStruct;
-import software.amazon.jsii.tests.calculator.DiamondInheritanceTopLevelStruct;
-import software.amazon.jsii.tests.calculator.DoNotOverridePrivates;
-import software.amazon.jsii.tests.calculator.DoubleTrouble;
-import software.amazon.jsii.tests.calculator.EnumDispenser;
-import software.amazon.jsii.tests.calculator.EraseUndefinedHashValues;
-import software.amazon.jsii.tests.calculator.EraseUndefinedHashValuesOptions;
-import software.amazon.jsii.tests.calculator.GiveMeStructs;
-import software.amazon.jsii.tests.calculator.GreetingAugmenter;
-import software.amazon.jsii.tests.calculator.IFriendlier;
-import software.amazon.jsii.tests.calculator.IFriendlyRandomGenerator;
-import software.amazon.jsii.tests.calculator.IInterfaceWithProperties;
-import software.amazon.jsii.tests.calculator.IPublicInterface;
-import software.amazon.jsii.tests.calculator.IRandomNumberGenerator;
-import software.amazon.jsii.tests.calculator.InbetweenClass;
-import software.amazon.jsii.tests.calculator.IInterfaceImplementedByAbstractClass;
-import software.amazon.jsii.tests.calculator.JSObjectLiteralForInterface;
-import software.amazon.jsii.tests.calculator.JSObjectLiteralToNative;
-import software.amazon.jsii.tests.calculator.JSObjectLiteralToNativeClass;
-import software.amazon.jsii.tests.calculator.JavaReservedWords;
-import software.amazon.jsii.tests.calculator.JsiiAgent;
-import software.amazon.jsii.tests.calculator.Multiply;
-import software.amazon.jsii.tests.calculator.Negate;
-import software.amazon.jsii.tests.calculator.NodeStandardLibrary;
-import software.amazon.jsii.tests.calculator.NullShouldBeTreatedAsUndefined;
-import software.amazon.jsii.tests.calculator.NullShouldBeTreatedAsUndefinedData;
-import software.amazon.jsii.tests.calculator.NumberGenerator;
-import software.amazon.jsii.tests.calculator.OptionalStruct;
-import software.amazon.jsii.tests.calculator.PartiallyInitializedThisConsumer;
-import software.amazon.jsii.tests.calculator.Polymorphism;
-import software.amazon.jsii.tests.calculator.Power;
-import software.amazon.jsii.tests.calculator.PublicClass;
-import software.amazon.jsii.tests.calculator.ReferenceEnumFromScopedPackage;
-import software.amazon.jsii.tests.calculator.ReturnsPrivateImplementationOfInterface;
-import software.amazon.jsii.tests.calculator.StableStruct;
-import software.amazon.jsii.tests.calculator.Statics;
-import software.amazon.jsii.tests.calculator.StructWithJavaReservedWords;
-import software.amazon.jsii.tests.calculator.Sum;
-import software.amazon.jsii.tests.calculator.SyncVirtualMethods;
-import software.amazon.jsii.tests.calculator.UnionProperties;
-import software.amazon.jsii.tests.calculator.UsesInterfaceWithProperties;
-import software.amazon.jsii.tests.calculator.VariadicMethod;
+import software.amazon.jsii.tests.calculator.*;
```

Comment:
yeah!

Reply:
That was done automagically by my IDE... All hail to our IDE overlords!